test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperPartialExtensionsTest.cs

```diff
@@ -141,6 +141,79 @@ public class HtmlHelperPartialExtensionsTest
         }
 
         // Func<IHtmlHelper, IHtmlContent>, expected Model, expected ViewDataDictionary
+        public static TheoryData<Action<IHtmlHelper>, object, ViewDataDictionary> RenderPartialExtensionMethods
+        {
+            get
+            {
+                var viewData = new ViewDataDictionary(new EmptyModelMetadataProvider());
+                var model = new object();
+                return new TheoryData<Action<IHtmlHelper>, object, ViewDataDictionary>
+                {
+                    { helper => helper.RenderPartial("test"), null, null },
+                    { helper => helper.RenderPartial("test", model), model, null },
+                    { helper => helper.RenderPartial("test", viewData), null, viewData },
+                };
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(RenderPartialExtensionMethods))]
+        public void RenderPartialMethods_DoesNotWrapThrownException(
+            Action<IHtmlHelper> partialMethod,
+            object unusedModel,
+            ViewDataDictionary unusedViewData)
+        {
+            // Arrange
+            var expected = new InvalidOperationException();
+            var helper = new Mock<IHtmlHelper>();
+            helper.Setup(h => h.RenderPartialAsync("test", It.IsAny<object>(), It.IsAny<ViewDataDictionary>()))
```

Comment:
I think the `Callback` is indented to align with `Setup`. Instead, just hit tab. We've been fixing existing code where we went out of the way to keep the `.`s aligned.